db/seeds.rb

```diff
@@ -6,3 +6,7 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+5.times do
+    teacher = Teacher.create!(first_name: Faker::Name.first_name, last_name: Faker::Name.last_name,
+        email: Faker::Internet.email, password: Faker::Internet.password(min_length: 6))
```

Comment:
Two things here :) 

The main purpose of having seeds is being able to do something with the records created by seeds. So, if your seeds create records for a user/teacher, then I expect to be able to login with the credentials that I'll find in the seeds. `Faker::Internet.email` and `Faker::Internet.password` are not really helpful to me in this case 😄 Sure, I can do a DB query to find out email, but how will I find out password to log in? 😄 In other words: try to make seeds usable for future you and future me :) 

Second thing: formatting 😄 This is purely subjective matter, I usually prefer this kind of style of dealing with long lines:

```suggestion
    teacher = Teacher.create!(
      first_name: Faker::Name.first_name,
      last_name: Faker::Name.last_name,
      email: Faker::Internet.email,
      password: Faker::Internet.password(min_length: 6)
    )
```

Idk, it just reads better to me 😅 However, if you're ok with the current style, feel free to continue using it - I've seen people who prefer this too :)

Reply:
Yes, your syntax is much better. I hope what I've commited is what You've meant.